### PR TITLE
#164198498 Follow, Get and Unfollow users

### PR DESCRIPTION
**Pull request by shaolinmkz**
#### Description of Task to be completed?
Having the following endpoint functional:
* GET  `host:port/api/profiles/followers`
* GET  `host:port/api/profiles/following`
* POST `host:port/api/profiles/:username/follow`
* DELETE `hostport/api/profiles/:username/follow`

#### How should this be manually tested?
1. Clone this branch.
2. Run `npm install` to install the dependencies.
3. Run `npm test` to run the tests to see test results.
4. Run `npm run start:dev` to test manually with Postman.

#### Any background context you want to provide?
* I created a task to capture have users view who follows them and who they are following

#### What are the relevant pivotal tracker stories?
[#164198498](https://pivotaltracker.com/story/show/164198498)

### CODE REVIEW

File: server/middlewares/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import AuthenticateUser from './AuthenticateUser';
+
+export { AuthenticateUser };
```
[the remark this answers is not in the document]
Comment (andela-gike) [reply server/middlewares/index.js:3]: Could you resolve this eslint error

**Review by andela-gike — COMMENTED** [review]
Aside from the comment I left you, the PR look good

**Comment by Dsalz** [comment]
looks good

**Review by Dsalz — APPROVED** [review]
looks good

**Pull request merged**